<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RingVacuumWindow</class>
 <widget class="QMainWindow" name="RingVacuumWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SR Vacuum</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string>Injector</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="ltBooster">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string>TL2</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="ltTL2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <widget class="QwtPlot" name="plotIMG">
        <property name="minimumSize">
         <size>
          <width>900</width>
          <height>300</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QwtPlot" name="plotIP">
        <property name="minimumSize">
         <size>
          <width>900</width>
          <height>300</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string>Storage-Ring</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="ltCells1">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="ltCells2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="titleGroupBox_2">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
 	background-color: rgb(255, 255, 255);
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Gauges</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnIonPumps">
         <property name="text">
          <string>Ion-Pumps</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnValves">
         <property name="text">
          <string>Valves</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnHeatAbsorbers">
         <property name="text">
          <string>Heat Absorbers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSetpoints">
         <property name="text">
          <string>IMG Setpoints</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>27</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
